ast/Trees.h

```diff
@@ -590,7 +590,7 @@ class ConstantLit final : public Expression {
 private:
     virtual void _sanityCheck();
 };
-// CheckSize(ConstantLit, 40, 8);
+CheckSize(ConstantLit, 56, 8);
```

Comment:
This is from the time when all ConstLits were uniformily resolved(and always to a symbol)